### ISSUE TITLE
[Accessibility] Support Home and End keys in Open Detected Link view

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#171918]
CC @meganrogge and @isidorn

## Reproducible Steps

1. Enable accessibility mode.

1. go to terminal input area.

1. Press Ctrl+Shift+O to open detected links.

1. Press either Home or End key to see if either puts you to the first or last item: It does not work.

## Expected Behavior

In the long list of detected links, Home and End key would be useful for quick navigation.

Home and End key works in `workbench.action.quickOpenView` (Ctrl+Q).

## System Info

Version: 1.75.0-insider (system setup)
Commit: 9ccc2b3b30c122afe45bc6722bef0e3901fb2806
Date: 2023-01-20T05:23:24.298Z
Electron: 19.1.9
Chromium: 102.0.5005.194
Node.js: 16.14.2
V8: 10.2.154.23-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by meganrogge** [comment]
`workbench.action.quickOpenView ` is a list, so works a bit differently. You can press down arrow to go to the first one or up arrow to go to the last one.

**Comment by jooyoungseo** [comment]
Ah ha! It makes sense for using UpArrow to go to the last item. Thanks for sharing the tip, @meganrogge !
